JitsiConference.js

```diff
@@ -2105,10 +2113,6 @@ JitsiConference.prototype._onIceConnectionRestored = function(session) {
     } else {
         this.isJvbConnectionInterrupted = false;
     }
-
-    if (session.isP2P === this.isP2PActive()) {
-        this.eventEmitter.emit(JitsiConferenceEvents.CONNECTION_RESTORED);
-    }
```

Comment:
Why was this bit removed?

Reply:
Oh, I think that might have been a mistake. Will revert